### DISCUSSION TITLE
Have all event-reminders go to #vc-events

### DISCUSSION THREAD

**Discussion by ebanner** [Virtual-Coffee/virtualcoffee.io#1157] [New Ideas]
I noticed the 10 minute reminder for Frontend Friday Folks Fighting CSSBattle.dev only went to #dev-frontend

I think it would be nice if all event reminders when to one place (`#vc-events`). Then optionally I think it would be nice/cool/make sense for the event to also go to a more specific channel (e.g. `#dev-frontend`)

My thinking is the video events are so central to the VC experience that I don't want the event reminders to be missed if someone didn't happen to read a new unread message in a specific channel

**Comment by danieltott** [comment]
This was how it used to be - but there was just too much going on. The announcements channel has a daily post with all of the events and where to find them - that's were we landed at the moment. Would be curious if anyone else would prefer them all to just be in events or in the specific channels

**Comment by BekahHW** [comment]
I agree that having them all go to #vc-events channel makes it more accessible. Before, when we had them in announcements, it was hard to see community-related announcements for things like the lightning talks. On Apr 13, 2024, at 12:16 PM, Dan Ott ***@***.***> wrote:﻿
This was how it used to be - but there was just too much going on. The announcements channel has a daily post with all of the events and where to find them - that's were we landed at the moment. Would be curious if anyone else would prefer them all to just be in events or in the specific channels

**Comment by ebanner** [comment]
I'm interested in making all reminders go to `#vc-events`

Last Friday for CSSBattles, I instinctively went to `#vc-events` expecting the 10 minute reminder link to be there. I actually clicked on the latest link there without even thinking and wound up waiting in a zoom room for a past event, not noticing it wasn't the right event

Having all reminders go to one place is the way to go IMO. I want one central place where I can go to access a link for an event I know is about to happen/is happening

Having a single visual indicator which means "yes there is an event happening/about to happen" is important (seeing a new unread message in the sidebar for `#vc-events`). Whereas if I see a new unread message in the sidebar for `#dev-frontend`, that could be any old message (not necessarily an event) 🤷‍♂️

If folks also want the Frontend Friday event to go to `#dev-frontend`, I'm fine with that. But I'm feeling strongly it should go to `#vc-events` as its primary destination

**Comment by danieltott** [comment]
The leaders can set these right now - @ebanner why don't you talk to @Narigo and see what he thinks? He's able to change the notification channel in the sidebar.

It would probably be useful to see what other group leaders have to say on the matter.

**Reply by ebanner** [reply]
Hey Dan, I chatted with Jorn last week about it during Frontend Friday. He seemed to be generally on board with switching it to #vc-events.

I'll reach out to him and confirm though.

**Reply by ebanner** [reply]
FYI Meg also seems to be on board as well

https://github.com/Virtual-Coffee/cms.virtualcoffee/issues/11

**Reply by ebanner** [reply]
Separately, I think I like having separate #announcements and #vc-events channels

I think it makes sense given what you guys said about it being too cluttered everything in one channel based on what you guys said

**Comment by Narigo** [comment]
I think it would be best to have all event infos in one channel. Maybe we can use different emojis to make it pop out more if this is some recurring event or a special lunch and learn or the regular coffee talks?

For the CSS battles, I remember in the beginning we used to have them in one channel and people would just find them. We also used to invite people by mentioning them directly. I don't know how much that made us have a bigger group in the past.

**Reply by ebanner** [reply]
@Narigo Hm, if you would like everything to go to one channel, could you open up a separate discussion for that?

I'd like to keep this discussion focused narrowly on whether the Frontend Fridays reminder should go to `#vc-events` (along with every other existing reminder/link) or whether it should go to its own separate channel (in `#dev-frontend`).

**Comment by ebanner** [comment]
Ok I'm definitely on board now with Meg's proposal

https://github.com/Virtual-Coffee/cms.virtualcoffee/issues/11

**Comment by ebanner** [comment]
FYI I've got a PR open for this

https://github.com/Virtual-Coffee/webhooks/pull/24
